backend/src/controllers/bars.ts

```diff
@@ -42,16 +53,16 @@ const createBarController = async (
   }
   //check if the beverages categories does exist
   let shouldContinue = true;
-  if(req.body.beverages != undefined){
-    let arrayOfBeverages : [beverage.Beverage] = req.body.beverages;
-    arrayOfBeverages.forEach(e => {
-      if(!verifyBeverageCategory(e.category)) {
-        shouldContinue=false;
+  if (req.body.beverages != undefined) {
```

Comment:
Tu veux pas faire un if/else plutot que d'avoir une variable should continue? Je vois pas trop son intérêt vu qu'on peut faire la même chose avec if/else et return dans le if si tu veux mettre fin à la fct

Reply:
Ouais je peux tout à fait faire un if else, je trouvais juste ça moins lisible (perso j'aime pas qd y'a trop de if else les uns dans les autres)